modules/openy_gc_auth/modules/openy_gc_auth_custom/openy_gc_auth_custom.info.yml

```diff
@@ -11,3 +11,4 @@ dependencies:
   - drupal:recaptcha
   - drupal:rest
   - drupal:openy_gc_auth
+  - drupal:openy_upgrade_tool
```

Comment:
hmmmm

Reply:
**openy_upgrade_tool** service used in hook update...

Reply:
Ok